### ISSUE TITLE
New desktop device via mobile device, prompts for paper key

### ISSUE THREAD

**Issue by amillerchip** [keybase/keybase-issues#2977]
https://keybase.io/adamu/graph

When I created `Bronn` by scanning a QR code from `Tyrion`, I was prompted by the desktop client to enter the `current dune` key for rekeying. I cancelled the prompt, and it popped up a few more times, until it eventually stopped.

Is it necessary to enter the paper key? I thought the link from the mobile device was enough. Is this left over from before mobile devices were supported?

**Comment by fiatjaf** [comment]
You can use either another device you have already registered (it can be a mobile device or a desktop) or a paper key to add any new device.

**Comment by amillerchip** [comment]
Yes, I was using a desktop device to create a mobile device. But why did the desktop device prompt for a paper key?

**Comment by maxtaco** [comment]
Cc: @chrisnojima .  This is a bug that we haven't fixed yet.  @amillerchip was the prompt for "paper key" because of keying problems?  It likely goes away on its own...

**Comment by amillerchip** [comment]
I don't know why it was prompting. As I said above, after repeatedly cancelling the prompt it stopped working. The new device was added fine.
